umbraco/Umbraco-CMS
# Fix delivery API cache level for media picker property editors

### Prerequisites


### Description

Media picker properties are being cached a bit too hard for the Delivery API, to the point where media changes are not reflected in the API output before a restart.

This PR adjusts these cache leves.

### Testing this PR

1. Create content with a picked media. It doesn't matter if it's MediaPicker or MediaPicker3.
2. Request the content using the Delivery API.
3. Change the picked media name.
4. Request the content again and verify that the media name is updated in the API repsonse.

If you want to get fancy, also test this with custom media properties (like an ALT text) using output expansion 😄
